core/app/modules/onboarding/assets/js/components/layout/header.scss

```
@@ -36,6 +36,20 @@
 		.eicon-pro-icon {
 			color: $brand-color-dark-pink;
 		}
+
+		&.onboarding-go-pro__btn {
```

Why not something like this?
```suggestion
		&__go-pro-btn {
```